enterprise/frontend/src/metabase-enterprise/whitelabel/index.js

```diff
@@ -24,265 +18,17 @@ import {
   getShowMetabaseLinks,
 } from "metabase-enterprise/settings/selectors";
 
-import ColorSettingsWidget from "./components/ColorSettingsWidget";
-import FontFilesWidget from "./components/FontFilesWidget";
-import FontWidget from "./components/FontWidget";
-import { HelpLinkSettings } from "./components/HelpLinkSettings";
-import { IllustrationTitle } from "./components/IllustrationTitle";
-import { IllustrationWidget } from "./components/IllustrationWidget";
-import { ImageUpload } from "./components/ImageUpload";
 import { LandingPageWidget } from "./components/LandingPageWidget";
 import LogoIcon from "./components/LogoIcon";
-import { MetabotToggleWidget } from "./components/MetabotToggleWidget";
-import {
-  MetabaseLinksToggleDescription,
-  SwitchWidget,
-} from "./components/SwitchWidget";
-import { getLoadingMessageOptions } from "./lib/loading-message";
+import { WhiteLabelSettingsPage } from "./components/WhiteLabelSettingsPage";
 import { updateColors } from "./lib/whitelabel";
 
 if (hasPremiumFeature("whitelabel")) {
   PLUGIN_LANDING_PAGE.getLandingPage = () =>
     MetabaseSettings.get("landing-page");
   PLUGIN_LANDING_PAGE.LandingPageWidget = LandingPageWidget;
-  PLUGIN_ADMIN_SETTINGS_UPDATES.push((sections) => ({
-    ...sections,
-    whitelabel: {
-      name: t`Appearance`,
-      key: "-redirect-to-branding",
-      settings: [
-        {
-          key: "-redirect-to-branding",
-          widget: () => (
-            <RedirectWidget to="/admin/settings/whitelabel/branding" />
-          ),
-        },
-      ],
-    },
```

Comment:
This is very silly, we've got a bunch of fake settings here for the sake of inserting redirects and extra titles and text. In the new structure, you can just put components in a page. ☺️